tableauserverclient/_version.py

```diff
@@ -84,7 +84,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
                 stderr=(subprocess.PIPE if hide_stderr else None),
             )
             break
-        except EnvironmentError:
+        except OSError:
```

Comment:
Non f-string change, but according to the [Built-in Exceptions docs](https://docs.python.org/3/library/exceptions.html#OSError.filename2) `EnvironmentError` was merged into `OSError` in 3.3.